apache/superset
# fix: Dates alignment in Table viz

### SUMMARY

Fixes #11823

This PR upgrades the Inter font library and applies the `tnum` property.

### AFTER

<img width="1058" alt="Screen Shot 2022-04-12 at 20 34 48" src="https://user-images.githubusercontent.com/60598000/163020945-5b8d83c6-391f-41b9-8dcc-642f1f31848a.png">

### TESTING INSTRUCTIONS
1. Open a Table viz
2. Group by a date column
3. Make sure dates are all vertically aligned

### ADDITIONAL INFORMATION


- [x] Has associated issue: #11823
